sentry-rails/Gemfile

```diff
@@ -12,47 +12,51 @@ platform :jruby do
   gem "jdbc-sqlite3"
 end
 
+ruby_version = Gem::Version.new(RUBY_VERSION)
+
 rails_version = ENV["RAILS_VERSION"]
-rails_version = "7.1.0" if rails_version.nil?
+rails_version = "7.2.0" if rails_version.nil?
 rails_version = Gem::Version.new(rails_version)
 
-if rails_version < Gem::Version.new("6.0.0")
-  gem "sqlite3", "~> 1.3.0", platform: :ruby
-else
-  if Gem::Version.new(RUBY_VERSION) >= Gem::Version.new("2.7.0")
-    gem "sqlite3", "~> 1.7.3", platform: :ruby
-  else
-    gem "sqlite3", "~> 1.6.9", platform: :ruby
-  end
-end
+gem "rails", "~> #{rails_version}"
 
 if rails_version >= Gem::Version.new("8.0.0.alpha")
-  gem "rails", github: "rails/rails"
   gem "rspec-rails"
+  gem "sqlite3", platform: :ruby
 elsif rails_version >= Gem::Version.new("7.1.0")
-  gem "rails", "~> #{rails_version}"
   gem "rspec-rails"
+  gem "sqlite3", "~> 1.7.3", platform: :ruby
+elsif rails_version >= Gem::Version.new("6.1.0")
+  gem "rspec-rails", "~> 4.0"
+
+  if ruby_version >= Gem::Version.new("2.7.0")
+    gem "sqlite3", "~> 1.7.3", platform: :ruby
+  else
+    gem "sqlite3", "~> 1.6.9", platform: :ruby
+  end
 else
   gem "rspec-rails", "~> 4.0"
-  gem "rails", "~> #{rails_version}"
   gem "psych", "~> 3.0.0"
-end
-
-gem "mini_magick"
-
-gem "sprockets-rails"
-
-gem "sidekiq"
-
 
-ruby_version = Gem::Version.new(RUBY_VERSION)
+  if rails_version >= Gem::Version.new("6.0.0")
+    gem "sqlite3", "~> 1.4.0", platform: :ruby
+  else
+    gem "sqlite3", "~> 1.3.0", platform: :ruby
+  end
+end
 
 if ruby_version < Gem::Version.new("2.5.0")
   # https://github.com/flavorjones/loofah/pull/267
   # loofah changed the required ruby version in a patch so we need to explicitly pin it
   gem "loofah", "2.20.0"
 end
 
+gem "mini_magick"
+
+gem "sprockets-rails"
+
+gem "sidekiq"
```

Comment:
@st0012 we probably no longer need this since test adapter is used for AJ tests

Reply:
Good catch 👍 Removed